Blockzilla/TelemetryIntegration.swift

```diff
@@ -43,14 +40,12 @@ class TelemetryEventObject {
     public static let app = "app"
     public static let searchBar = "search_bar"
     public static let eraseButton = "erase_button"
-    public static let onboarding = "new_onboarding_experience"
-    public static let firstRun = "previous_first_run_experience"
+    public static let onboarding = "new_onboarding"
```

Comment:
I don't see either of these in the android code base.

Reply:
Ours has to be named differently since we are supporting two onboardings right now

Reply:
The weird thing is in the tables @bbinto has shown us, we _are_ seeing the 'new_onboarding', and not "new_onboarding_experience" - any idea why? Honestly, I would rather you name this something ios specific, like 'ios_onboarding_v1'.